lib/views/pages/transactions/add_expense.dart

```diff
@@ -24,33 +28,36 @@ class _AddExpenseFormState extends State<AddExpenseForm> {
     'dateController': TextEditingController(),
   };
 
-  //TODO: change dropdown values here
-  List<String> dropdownValues = [
-    '',
-    'Food and Drink',
-    'Bills',
-    'Other',
-  ];
-
-  onSubmit() {
+  onSubmit(Expense? snapshot) async {
     if (expenseFormKey.currentState!.validate()) {
-      //TODO: Add implementation for submission of form
-      showDialog(
-        context: context,
-        builder: (BuildContext context) => AlertDialog(
-          content: Column(
-            mainAxisSize: MainAxisSize.min,
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              Text('Name: ${formInputControllers['nameController']!.text}'),
-              Text('Amount: ${formInputControllers['amountController']!.text}'),
-              Text(
-                  'Category: ${formInputControllers['categoryController']!.text}'),
-              Text('Date: ${formInputControllers['dateController']!.text}'),
-            ],
-          ),
-        ),
-      );
+      final createdAt = DateTime.now();
+      final updatedAt = DateTime.now();
+
+      Expense expense = Expense();
+      expense.category = formInputControllers['categoryController']!.text;
```

Comment:
**We need to save integer values, not string**
<img width="334" alt="image" src="https://user-images.githubusercontent.com/20507565/165506683-9c3d46d4-b0f8-4303-9025-518e685566f7.png">

Its better to set enums like this,
```
enum CategoryEnum {
  food,
  transportation,
  apparel,
  health,
  social,
}
```

We can fetch its index values, so we can generate map objects like this
```
CategoryEnum.values.map((e) => {e.index: e.name.capitalize()})
```
Output of above sample code:  (can add `+ 1`, to start at `1` and not `0`)
<img width="717" alt="image" src="https://user-images.githubusercontent.com/20507565/165506985-bab4436d-c49b-485a-a19f-b7f60ea80e9e.png">